pkg/apiserver/event/event.go

```diff
@@ -47,11 +52,52 @@ func NewService(
 
 // Register mounts our HTTP handler on the mux.
 func Register(r *gin.RouterGroup, s *Service) {
-	endpoint := r.Group("/event")
+	endpoint := r.Group("/events")
 
 	// TODO: add more api handlers
 	endpoint.GET("/all", s.listEvents)
 }
 
-// TODO: need to be implemented
-func (s *Service) listEvents(c *gin.Context) {}
+// @Summary Get all events from db.
+// @Description Get all chaos kinds from db.
+// @Tags events
+// @Produce json
+// @Success 200 {array} string
+// @Router /api/events/all [get]
+// @Failure 500 {object} utils.APIError
+func (s *Service) listEvents(c *gin.Context) {
+	name := c.Query("name")
+	namespace := c.Query("namespace")
+	eventList := make([]*core.Event, 0)
+
+	if name == "" && namespace == "" {
```

Comment:
Did we need to define a funtion like CheckNameAndNamespace(It can be done in other PR)